hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -44,23 +54,70 @@
           " of the entire cluster) no more than the threshold value.")
   private String threshold = "0.1";
 
-  @Config(key = "datanodes.balanced.max", type = ConfigType.INT,
-      defaultValue = "5", tags = {ConfigTag.BALANCER}, description = "The " +
-      "maximum number of datanodes that should be balanced. Container " +
-      "Balancer will not balance more number of datanodes than this limit.")
-  private int maxDatanodesToBalance = 5;
+  @Config(key = "datanodes.involved.max.per.iteration", type = ConfigType.AUTO,
```

Comment:
NIT: rename `max` to `ratio` or sth better.

Reply:
Done